### PR TITLE
Update async.md

### PR DESCRIPTION
**Pull request by jslavin-clearblade**

### CODE REVIEW

File: async.md [hunk 1]
```diff
@@ -428,9 +428,9 @@ ClearBladeAsync.File(deployment_name, path)
 File.stat()
 
 /**
- * Reads the entire contents of the file.
- * The promise is resolved with the file contents.
- * If encoding is not specified, contents will be a UInt8Array.
+ * Reads the file's entire contents.
+ * The promise is resolved with them.
```
Comment (WorkForPizza) [inline async.md:432]: I disagree with your use of `them` here.

File: async.md [hunk 2]
```diff
@@ -1288,48 +1288,48 @@ Adapter.deleteFile(filename)
...
- * @property {string} output - the combined stdout and stderr output resulting from executing the shell command
+ * @property {string} err: Will contain an error if the control command fails
+ * @property {string} command: The shell command which got executed
+ * @property {string} output: The combined stdout and stderr output resulting from executing the shell command
  */
 
 /**
  * Sends a control command to the given edges.
- * Valid commands are: "start", "stop", "restart", "status", "undeploy", "logs".
+ * Valid commands are: start, stop, restart, status, undeploy, and logs.
```
Comment (WorkForPizza) [inline async.md:1298]: these are string literals, we should keep the quotes

File: async.md [hunk 3]
```diff
@@ -1288,48 +1288,48 @@ Adapter.deleteFile(filename)
...
  * The edges param is an array of edge names.
  * If run on the edge, the edges param may only include the current edge's name.
- * Promise will resolve with an object whose keys are edge names and values are AdapterControlResponses.
+ * Promise will resolve with an object whose keys are edge names whose and values are AdapterControlResponses.
```
Comment (WorkForPizza) [inline async.md:1301]: `whose` should be after `and` here, right?

File: async.md [hunk 4]
```diff
@@ -1288,48 +1288,48 @@ Adapter.deleteFile(filename)
...
  * Sends data to the destination at the specified interval
- * @param {string} collectionName - collection name of the data you are trying to sync
- * @param {Object} data - the actual item or items created/updated/deleted
- * @param {string} destination - can be CustomSync.Platform if on the edge or CustomSync.AllEdges if on the platform or an individual edge name if on the platform
- * @param {string} [interval=now] - optional but can be CustomSync.Now or be actual interval string values like “60s”, “10m”, “1h”, “5d”. Only seconds, minutes, hours and days are supported. If interval is not specified, CustomSync.Now is the default.
+ * @param {string} collectionName: Collection name of the data you are trying to sync
+ * @param {Object} data: The actual item or items created/updated/deleted
+ * @param {string} destination: Can be CustomSync.Platform if on the edge or CustomSync.AllEdges or an individual edge name if on the platform
+ * @param {string} [interval=now]: Optional but can be CustomSync.Now or be actual interval string values like 60s, 10m, 1h, and 5d. Only seconds, minutes, hours, and days are supported. If the interval is not specified, CustomSync.Now is the default.
```
Comment (WorkForPizza) [inline async.md:1326]: keep the quotes around string literals

File: async.md [hunk 5]
```diff
@@ -1549,24 +1549,24 @@ GoogleCloudMonitoring.reportSentBytesCountMetric(bytesCount)
...
 /**
- * @typedef {Object} AssetRow - a row in the assets collection
- * @property {string} asset_id - a unique identifier for each asset
- * @property {string} asset_type - can be "thermometer" or "barometer"
- * @property {number} reading - the current temperature or pressure value
- * @property {string} last_updated - timestamp when the asset last sent an update
- * @property {boolean} stale - indicates the asset hasn't sent an update in over a week
+ * @typedef {Object} AssetRow: A row in the assets collection
+ * @property {string} asset_id: Each asset's unique identifier
+ * @property {string} asset_type: Can be thermometer or barometer
```
Comment (WorkForPizza) [inline async.md:1560]: string literals - keep quotes

File: async.md [hunk 6]
```diff
@@ -1661,8 +1661,8 @@ function countAssetsByType() {
 }
 
 /**
- * findNewDevices runs a raw sql query against the device table
- * to find all devices created in the past 7 days whose type is in the allowedDeviceTypes list.
+ * findNewDevices runs a raw SQL query against the device table
+ * To find all devices created in the past seven days whose type is in the allowedDeviceTypes list.
```
Comment (WorkForPizza) [inline async.md:1665]: `To` should remain lowercase, it's not starting a new sentence

File: async.md [hunk 7]
```diff
@@ -1727,7 +1727,7 @@ function sendAdapterLogsToPlatform() {
 
 /**
  * checkAdapterLogsForErrors reads a log file from the platform's inbox,
- * and returns the lines from those logs that contain the string "ERROR".
+ * and returns the lines from those logs containing the ERROR string.
```
Comment (WorkForPizza) [inline async.md:1730]: string literal - keep quotes

File: async.md [hunk 8]
```diff
@@ -3,32 +3,32 @@ The `init` function is not required.
...
-1. [Adapters](#adapters)
-1. [Collection Custom Sync](#collection-custom-sync)
-1. [Secret](#secret)
-1. [Preloader](#preloader)
-1. [GoogleCloudLogger](#google-cloud-logger)
-2. [Data Usage](#data-usage)
-1. [GoogleCloudMonitoring](#google-cloud-monitoring)
+2. [Collections](#collections)
+3. [Databases](#databases)
+4. [File management](#file-management)
```
[the remark this answers is not in the document]
Comment (jslavin-clearblade) [reply async.md:8]: They can be in sentence case. It's more informal.

**Review by WorkForPizza — APPROVED** [review]

**Pull request merged**